ebpf-for-windows.sln

```diff
@@ -797,7 +797,6 @@ Global
 		{21236F4F-341C-454F-B55B-C653897AB3E8}.NativeOnlyDebug|ARM64.ActiveCfg = Debug|x64
 		{21236F4F-341C-454F-B55B-C653897AB3E8}.NativeOnlyDebug|ARM64.Build.0 = Debug|x64
 		{21236F4F-341C-454F-B55B-C653897AB3E8}.NativeOnlyDebug|x64.ActiveCfg = Debug|x64
-		{21236F4F-341C-454F-B55B-C653897AB3E8}.NativeOnlyDebug|x64.Build.0 = Debug|x64
```

Comment:
I don't follow why these lines were removed.

Reply:
These are projects that are being built for as Debug during the NativeOnlyBuild and hence are not being uploaded during the build process (i.e they are unused).

The issue is that the generation of the version header places it in $(OutDir), which is different for Debug vs NativeOnlyDebug. 

As a result, these fail to build. Given that they aren't actually ever being used, I removed them from the build rather than fix them.